distributions/Dist_dl3185.py

```diff
@@ -0,0 +1,47 @@
+
+import numpy as np
+from base_distribution import BaseDistribution
+
+class Dist_dl3185(BaseDistribution):
+	def __init__(self):
+		self.f_max = 1
+		self.x_min = -1
+		self.x_max = 1
+
+
+	def pdf(self, x):
+		"""This is your PDF"""
+		return x**2
+
+	def mean(self):
+		"""This is the mean of the PDF"""
+		return 0
+
+	def std(self):
+		"""This is the standard deviation of the pdf"""
+		return np.sqrt(0.4)
```

Comment:
where does 0.4 come from? It would be nicer to have an equation here based on your pdf, x_min, and x_max